### PR TITLE
Performance analytics

### PR DESCRIPTION
**Pull request by holiber**
Run performance tests and save results to DB

### CODE REVIEW

File: app/services/metrics/metrics.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * An utility service that keeps the time of some key events to help find performance issues
+ * This services is used in performance tests
+ */
+export class MetricsService extends Service {
+  @Inject() performanceService: PerformanceService;
+
+  protected init() {
+    const appStarTime = ipcRenderer.sendSync('getAppStartTime');
```
Comment (andycreeth) [inline app/services/metrics/metrics.ts:20]: Should we be disabling this service in production?  Don't want to be a doing a synchronous IPC on startup in production for no reason.

File: package.json [hunk 2]
```diff
@@ -65,6 +67,7 @@
     "@types/node-fetch": "^2.5.5",
     "archiver": "2.1.1",
     "aws-sdk": "^2.344.0",
+    "cli-table": "^0.3.1",
```
Comment (andycreeth) [inline package.json:70]: This should be under `devDependencies`

File: test/helpers/repo.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  // fetching the commit SHA
+  const lastCommits = execSync('git log -n 2 --pretty=oneline')
+    .toString()
+    .split('\n')
+    .map(log => log.split(' ')[0]);
+
+  // the repo is in the detached state for CI
+  // we need to take a one commit before to take a commit that has been associated to the PR
+  return CI ? lastCommits[1] : lastCommits[0];
+}
```
Comment (andycreeth) [inline test/helpers/repo.js:29]: FYI recently learned about `git rev-parse` during my bundle work.  Can do nifty stuff like `git rev-parse HEAD` or `git rev-parse HEAD^` to get the commit SHA of HEAD, or the commit behind HEAD.

Comment (holiber) [reply test/helpers/repo.js:29]: Nice, will use that

**Pull request merged**